tests/PhpBrew/Command/KnownCommandTest.php

```diff
@@ -32,6 +32,7 @@ public function testMoreOption()
 
     /**
      * @outputBuffering enabled
+     * @group mayignore
```

Comment:
Add the `mayignore` group to skipp HTTP interaction test during GitHub action running.